2-data-model/data-modeling-in-rethinkdb.md

```diff
@@ -65,9 +65,7 @@ __Advantages of using embedded arrays:__
 
 __Disadvantages of using embedded arrays:__
 
-- Any operation on an author document requires loading all posts into
-  memory. Any update to the document requires rewriting the full array
-  to disk.
+- Any operation on a document in the `author` table requires loading all posts into memory. Note that any update to a document requires rewriting the entire document to disk. 
```

Comment:
Actually this formulation sounds a bit weird because it talks about "any update to a document" and then about "the entire document". Maybe the first one should be "to an embedded document"?

I'm not sure about the "all posts" question. It might be obvious from the context that this only includes posts by the given author. How about "requires loading all associated posts into memory"?